.changeset/old-rockets-doubt.md

```diff
@@ -0,0 +1,11 @@
+---
+'@backstage/backend-common': minor
```

Comment:
Realized this should be a `patch` change as it's non-breaking and the package is at 0.x

Reply:
okay I change it.

Reply:
Done!